gensrc/thrift/PlanNodes.thrift

```diff
@@ -899,6 +903,56 @@ struct TConnectorScanNode {
   // 2: optional THdfsScanNode hdfs_scan_node
 }
 
+// TODO
+struct TBinlogSourceNode {
+}
+
+struct TStreamJoinNode {
+  1: required TJoinOp join_op
+
+  // anything from the ON, USING or WHERE clauses that's an equi-join predicate
+  2: required list<TEqJoinCondition> eq_join_conjuncts
+
+  // anything from the ON or USING clauses (but *not* the WHERE clause) that's not an
+  // equi-join predicate
+  3: optional list<Exprs.TExpr> other_join_conjuncts
+  4: optional bool is_push_down
+  
+  // for profiling
+  21: optional string sql_join_predicates
+  22: optional string sql_predicates
+
+  52: optional TJoinDistributionMode distribution_mode;
+  53: optional list<Exprs.TExpr> partition_exprs
+  54: optional list<Types.TSlotId> output_columns
+}
+
+struct TStreamAggregationNode {
+  1: optional list<Exprs.TExpr> grouping_exprs
+  // aggregate exprs. The root of each expr is the aggregate function. The
+  // other exprs are the inputs to the aggregate function.
+  2: required list<Exprs.TExpr> aggregate_functions
+
+  // Tuple id used for intermediate aggregations (with slots of agg intermediate types)
+  3: required Types.TTupleId intermediate_tuple_id
```

Comment:
for aggregate node, the intermediate_tuple_id and output_tuple_id are needn't

Reply:
fixed